Modules/FindCUDA.cmake

```diff
@@ -308,8 +322,20 @@
 #   CUDA_nvcuvid_LIBRARY  -- CUDA Video Decoder library.
 #                            Only available for CUDA version 3.2+.
 #                            Windows only.
+#   CUDA_KNOWN_GPU_ARCH_NAMES      -- "Fermi" "Kepler" "Maxwell" "All" "Common" "Manual" "Auto"
```

Comment:
Is it possible to follow the format of the other variables' documentation?

These are values provided by the script, and only if you call something specific, right?